eclair-core/src/main/resources/reference.conf

```diff
@@ -358,7 +358,12 @@ eclair {
     # Transient connections opened to relay messages will be closed after this delay of inactivity
     kill-transient-connection-after = 30 seconds
 
-    max-per-peer-per-second = 10
+    rate-limit-per-peer {
+      number = 10
+      per = 1 second
+    }
+    max-per-peer = 10
```

Comment:
Why do you need to separate `number` and `per`?
You use it with `1 second` everywhere, so can't we just keep the previous `max-per-peer-per-second`?

Reply:
I've switched back to `max-per-peer-per-second`. It gives us less flexibility but I guess it's fine.